controllers/front/payment.php

```diff
@@ -255,8 +255,8 @@ public function postProcess()
             if ($order instanceof \Pagantis\OrdersApiClient\Model\Order) {
                 $url = $order->getActionUrls()->getForm();
                 $orderId = $order->getId();
-                $sql = "INSERT INTO `" . _DB_PREFIX_ . "pagantis_order` (`id`, `order_id`)
-                     VALUES ('$cart->id','$orderId') 
+                $sql = "INSERT INTO `" . _DB_PREFIX_ . "pagantis_order` (`id`, `order_id`, `ps_order_id`)
```

Comment:
The field ps_order_id accept null, so it is not necessary fill the field in this case

Reply:
sure, deleted